layouts/css/base.styl

```diff
@@ -3,9 +3,11 @@
 // Base styles
 body
     box-sizing border-box
-    font 300 20px/1.5 'Source Sans Pro', Arial, sans-serif
-    color #000
+    font 400 15px/1.5 "source-sans-pro", sans-serif
+    color #333333
```

Comment:
I feel this *may* be taking too much contrast away for those who have issues regarding less contrasting text. See: https://github.com/nodejs/iojs.org/pull/366

Perhaps `#222`?